src/App.js

```diff
@@ -19,27 +21,30 @@ import { owners } from "./data/owners";
 import { pets } from "./data/pets";
 
 class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      employees,
-      owners,
-      pets,
-    };
-  }
+  
 
   render() {
-    const { employees, owners, pets } = this.state;
+    
     return (
       <div className="wrapper">
-        <Nav />
-        <Home employees={employees} owners={owners} pets={pets} />
-        <StaffList employees={employees} />
-        <PetsList pets={pets} />
-        <Footer />
+
+
+        <Switch>
+        <Redirect exact from="/pets" to="/pets/cats" />
```

Comment:
Since the Nav and footer is always rendering you can have the component called outside the switch statement and won't have to add it to each route.
```

return (
  <div>
    <nav/>
    <switch>
       ...
    </switch>
    <footer/>
  </div>
)
```